docs/examples/vectorfields.py

```diff
@@ -0,0 +1,493 @@
+# %% [markdown]
+# # Gaussian Processes for Vector Fields
```

Comment:
"GPs for modelling ocean velocity" is better. Might as well be upfront with what we are doing. Vector fields is a bit too general and could mean lots of things. Also change the title of the notebook

Reply:
I agree that it would be good to provide details on what is modelled here, but from the perspective of a random JaxGP user, "ocean velocity" is unlikely to be directly relevant, whereas it would be more informative to talk about modelling vector fields, so I would be in favour of keeping "vector fields" (or something along those lines) in the title.

Reply:
changed to Gaussian Processes for Vector Fields and Ocean Current Modelling